vehicle/raw_vehicle_cmd_converter/launch/raw_vehicle_converter.launch.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```

Comment:
Not sure about the version. It was 1.0 before. Probably only a minor thing

Reply:
I'm not sure about the version either, but ROS2 design schema is using 1.0 so I think we can keep it as 1.0.
https://design.ros2.org/articles/roslaunch_xml.html